### ISSUE TITLE
dependencies issue on Ubuntu Bionic

### ISSUE THREAD

**Issue by jonadem** [allan-simon/linthesia#90]
I tried to install the software but I was blocked very soon, I get a dependencies conflict.

The problematic package is `librtmidi-dev` on Ubuntu 18.04 "Bionic".

When I try to install it, I get :

    jona@jona-ThinkPad-T520:~/Softwares/linthesia$ sudo apt install librtmidi-dev
    Reading package lists... Done
    Building dependency tree       
    Reading state information... Done
    Some packages could not be installed. This may mean that you have
    requested an impossible situation or if you are using the unstable
    distribution that some required packages have not yet been created
    or been moved out of Incoming.
    The following information may help to resolve the situation:
    
    The following packages have unmet dependencies:
     librtmidi-dev : Depends: librtaudio-dev but it is not going to be installed
    E: Unable to correct problems, you have held broken packages.

My guess is that the issue lies in this chain : librtmidi-dev -> libjack-dev -> libjack0 >< jackd1 <- lmms
`lmms` is installed from the official repository (a lot of other softwares are depending on jackd1 AFAICT).
I also tried to install `librtmidi-dev:i386` but I get the same problem.

Can someone give me an hint ?

Thank you

**Comment by allanlegalstart** [comment]
what do you get when you try to install alone `librtaudio-dev` ?

**Comment by jonadem** [comment]
If I try to install librtaudio-dev, I get :

    jona@jona-ThinkPad-T520:~$ sudo apt install librtaudio-dev
    [sudo] password for jona: 
    Reading package lists... Done
    Building dependency tree       
    Reading state information... Done
    The following packages were automatically installed and are no longer required:
      calf-ladspa caps fonts-wine gstreamer1.0-plugins-base:i386
      libasn1-8-heimdal:i386 libasound2:i386 libasyncns0:i386
      libavahi-client3:i386 libavahi-common-data:i386 libavahi-common3:i386
      libbsd0:i386 libcairo2:i386 libcap2:i386 libcapi20-3:i386
      libcdparanoia0:i386 libcups2:i386 libdbus-1-3:i386 libdrm-amdgpu1:i386
      libdrm-intel1:i386 libdrm-nouveau2:i386 libdrm-radeon1:i386 libdrm2:i386
      libedit2:i386 libelf1:i386 libexif12:i386 libexpat1:i386 libffi6:i386
      libflac8:i386 libfontconfig1:i386 libfreetype6:i386 libgd3:i386 libgl1:i386
      libgl1-mesa-dri:i386 libgl1-mesa-glx:i386 libglapi-mesa:i386
      libglib2.0-0:i386 libglu1-mesa:i386 libglvnd0:i386 libglx-mesa0:i386
      libglx0:i386 libgmp10:i386 libgnutls30:i386 libgphoto2-6:i386
      libgphoto2-port12:i386 libgsm1:i386 libgssapi-krb5-2:i386
      libgssapi3-heimdal:i386 libgstreamer-plugins-base1.0-0:i386
      libgstreamer1.0-0:i386 libhcrypto4-heimdal:i386 libheimbase1-heimdal:i386
      libheimntlm0-heimdal:i386 libhogweed4:i386 libhx509-5-heimdal:i386
      libicu60:i386 libidn2-0:i386 libieee1284-3:i386 libjbig0:i386
      libjpeg-turbo8:i386 libjpeg8:i386 libk5crypto3:i386 libkeyutils1:i386
      libkrb5-26-heimdal:i386 libkrb5-3:i386 libkrb5support0:i386 liblcms2-2:i386
      libldap-2.4-2:i386 libllvm7:i386 liblo7 libltdl7:i386 libmpg123-0:i386
      libmxml-bin libmxml-dev libmxml1 libnettle6:i386 libodbc1:i386 libogg0:i386
      libopenal1:i386 libopus0:i386 liborc-0.4-0:i386 libosmesa6:i386
      libp11-kit0:i386 libpcap0.8:i386 libpciaccess0:i386 libpixman-1-0:i386
      libpng16-16:i386 libpulse0:i386 libroken18-heimdal:i386 librtmidi4
      libsamplerate0:i386 libsane1:i386 libsasl2-2:i386 libsasl2-modules:i386
      libsasl2-modules-db:i386 libsensors4:i386 libsndfile1:i386 libsndio6.1:i386
      libspeexdsp1:i386 libsqlite3-0:i386 libssl1.1:i386 libstdc++6:i386
      libstk-4.5.0 libtasn1-6:i386 libtheora0:i386 libtiff5:i386
      libunistring2:i386 libusb-1.0-0:i386 libv4l-0:i386 libv4lconvert0:i386
      libvisual-0.4-0:i386 libvorbis0a:i386 libvorbisenc2:i386 libwebp6:i386
      libwind0-heimdal:i386 libwine:i386 libwrap0:i386 libx11-6:i386
      libx11-xcb1:i386 libxau6:i386 libxcb-dri2-0:i386 libxcb-dri3-0:i386
      libxcb-glx0:i386 libxcb-present0:i386 libxcb-render0:i386 libxcb-shm0:i386
      libxcb-sync1:i386 libxcb1:i386 libxcomposite1:i386 libxcursor1:i386
      libxdamage1:i386 libxdmcp6:i386 libxext6:i386 libxfixes3:i386 libxi6:i386
      libxinerama1:i386 libxml2:i386 libxpm4:i386 libxrandr2:i386 libxrender1:i386
      libxshmfence1:i386 libxslt1.1:i386 libxxf86vm1:i386 lmms-common
      lmms-vst-server:i386 ocl-icd-libopencl1:i386 stk tap-plugins wine-stable
      wine32:i386 zynaddsubfx zynaddsubfx-data
    Use 'sudo apt autoremove' to remove them.
    The following additional packages will be installed:
      libasound2-dev libjack-dev libjack0 uuid-dev
    Suggested packages:
      libasound2-doc jackd1 librtmidi-dev
    The following packages will be REMOVED:
      libasound2-plugins:i386 libjack-jackd2-0 libjack-jackd2-0:i386
      librtmidi4:i386 lmms
    The following NEW packages will be installed:
      libasound2-dev libjack-dev libjack0 librtaudio-dev uuid-dev
    0 upgraded, 5 newly installed, 5 to remove and 3 not upgraded.
    Need to get 467 kB of archives.
    After this operation, 11.3 MB disk space will be freed.
    Do you want to continue? [Y/n] n
    Abort.

**Comment by allan-simon** [comment]
I think you can try to type "yes" and then install `librtmidi-dev` , looking to the output  it seems nothing harmful will be removed

**Comment by jonadem** [comment]
I don't want `lmms` to be removed. This is an end-user application and I want to keep it. On top of that, I have a clean `apt autoremove` if I don't install `librtmidi-dev`.

**Comment by allan-simon** [comment]
@jonadem , was this issue solved with a recent upgrade of ubuntu ?

**Comment by jonadem** [comment]
Yes, I was able to install both `lmms` and `linthesia` on Ubuntu 19.10 (Eoan Ermine). As a side note, I started with `linthesia`.
